### PR TITLE
GO_CMD for dockerized build

### PR DESCRIPTION
**Pull request by andrewpcone**
Addresses #1177

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -54,6 +54,7 @@ endif
 # For now just take the first one
 DOCKER_GOPATH_TOKS := $(subst :, ,$(DOCKER_GOPATH:v%=%))
 DOCKER_GOPATH_DIR = $(word 1,$(DOCKER_GOPATH_TOKS))
+@echo docker gopath: ${DOCKER_GOPATH_DIR}
```
Comment (kannon92) [inline makefile:57]: I'm unfamilar with this syntax in make.  What does this do?

**Comment by andrewpcone** [comment]
oops that's debug code sorry

File: makefile (hunk 1 above)
Comment (richscott) [reply makefile:57]: If it's not expected to be needed in the future, I'd suggest removing it.

**Review by kannon92 — APPROVED** [review]

**Comment by andrewpcone** [comment]
This isn't needed if go is added to the path of the host box, so I'm going to get rid of it

**Pull request closed without merging**